### PR TITLE
doc: report a vulnerability

### PR DESCRIPTION
**Pull request by schomatis**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+For *critical* bugs, please send an email to security@filecoin.org.
```
Comment (arajasek) [inline SECURITY.md:5]: Wanna set that in [email syntax](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses), or are you afraid of spam?

File: README.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Lotus is an implementation of the Filecoin Distributed Storage Network. For more
 
 For instructions on how to build lotus from source, please visit [https://docs.lotu.sh](https://docs.lotu.sh) or read the source [here](https://github.com/filecoin-project/lotus/tree/master/documentation).
 
+## Reporting a Vulnerability
+
+Please send an email to security@filecoin.org. See our [security policy](SECURITY.md) for more details.
```
Comment (arajasek) [inline README.md:13]: Is that gonna link to the SECURITY page correctly (i'm not sure)?

File: SECURITY.md (hunk 1 above)
Comment (schomatis) [reply SECURITY.md:5]: i'm afraid of the email syntax

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The bug reporting process differs between bugs that are critical and may crash the network, and others that are unlikely to cause problems if malicious parties know about it. For non-critical bugs, please simply file a GitHub [issue](https://github.com/filecoin-project/lotus/issues/new?template=bug_report.md). 
+
+Please try to provide a clear description of any bugs reported, along with how to reproduce the bug if possible. More detailed bug reports (especially those with a PoC included) will help us move forward much faster. Additionally, please avoid reporting bugs that already have open issues. Take a moment to search the issue list of the related GitHub repositories before writing up a new report.
```
[the remark this answers is not in the document]
Comment (schomatis) [reply SECURITY.md:9]: proof of concept

File: README.md (hunk 2 above)
Comment (schomatis) [reply README.md:13]: yes, if not fully qualified it defaults to the root of the repo

File: SECURITY.md (hunk 1 above)
Comment (schomatis) [reply SECURITY.md:5]: when rendered it already links to the email (i guess it's some github magic)

File: README.md (hunk 2 above)
Comment (corviato1) [reply README.md:13]: What is the usual response time for reporting a bug via that email?  Or is there a better place to post?

**Pull request merged**